src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -20,6 +20,7 @@
 
   private static final int EUROPA_INFOBOX_WIDTH = 300;
   private static final int DEFAULT_INFOBOX_WIDTH = 270;
+  private static final String TEMPLATE_TYPE_INFOBOX = "Infobox";
```

Comment:
It's unclear to me why the types are scattered between different classes. Please use one common name, see https://github.com/Wikia/selenium-tests/tree/master/src/test/java/com/wikia/webdriver/common/contentpatterns